pkg/template/types.go

```diff
@@ -45,8 +45,9 @@ type Params struct {
 	// Kubernetes components allow the passing of extra `docker run` and
 	// `command` arguments to image commands. This allows, for example,
 	// the addition of cloud provider extensions.
-	Kubernetes Kubernetes
-	Node       v1alpha1.ClusterNode
+	Kubernetes           Kubernetes
+	KVMWorkerHostVolumes []v1alpha1.KVMConfigSpecKVMNodeHostVolumes
```

Comment:
Even though `k8scloudconfig` is soon going to be KVM-only, I'd prefer to use a provider-independent type for this. This could be a slice of structs, but if we're only using `MountTag`, it could just be a slice of `string`s.

Reply:
ok

Reply:
fixed.